training/lora/example/redpajama-incite-chat-3b_merge.py

```diff
@@ -0,0 +1,18 @@
+import torch
+from peft import PeftModel, PeftConfig
+from transformers import AutoModelForCausalLM
+
+peft_model_path = 'outputs/redpajama-incite-chat-3b-sample-lowrank'
+
+config = PeftConfig.from_pretrained(peft_model_path)
+model = AutoModelForCausalLM.from_pretrained(
+    config.base_model_name_or_path,
+    return_dict=True,
+    device_map='auto')
+
+# Load the Lora model
+model = PeftModel.from_pretrained(model, peft_model_path)
+
+model = model.merge_and_unload()
+
+model.save_pretrained('outputs/redpajama-incite-chat-3b-sample-lowrank/merge')
```

Comment:
I don't think it's a good idea to save the model in a directory inside the peft model output dir. Let's keep the two model outputs separate.
```suggestion
model.save_pretrained('outputs/redpajama-incite-chat-3b-sample-lowrank-merged')
```